composer.json

```diff
@@ -21,11 +21,11 @@
         "prooph"
     ],
     "require": {
-        "php": ">=5.5"
+        "php": ">=5.5",
+        "prooph/event-store" : "^5.1"
```

Comment:
Because this is a bridge between event-store and service-bus, the service bus should also be in required and not optional.